common/rpc/rpc.go

```diff
@@ -204,7 +204,7 @@ func (d *RPCFactory) CreateFrontendGRPCConnection(hostName string) *grpc.ClientC
 		}
 	}
 
-	return d.dial(hostName, tlsClientConfig)
+	return d.dial(hostName, tlsClientConfig, false)
```

Comment:
Nit:
(newish to golang, don't hesitate to tell me I'm wrong here, but ...)
I'd have written this code, just for readability:

keepAlive := false
return d.dial(hostName, tlsClientConfig, keepAlive)
... or even
return d.dial(hostName, tlsClientConfig, false) // keepAlive=false

Personally I think Golang should have included named args; I think it's one of the things python got right.